courses/free-courses-hi.md

```diff
@@ -540,6 +540,7 @@
 
 #### React
 
+* [🚀🔥 React & Redux Complete Course (2024) with Projects | Notes | Free Certification](https://www.youtube.com/watch?v=eILUmCJhl64&t=48567s) - Complete Coding by Prashant Sir
```

Comment:
```suggestion
* [React & Redux Complete Course (2024) with Projects](https://www.youtube.com/watch?v=eILUmCJhl64) - Complete Coding by Prashant Sir
```
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
no emojis